index.src.html

```diff
@@ -459,6 +465,140 @@
+    [=credential chooser=] popping up without context:
+
+    <pre>
+      window.addEventListener('load', _ =&gt; {
+        var c = await navigatior.<a for="Navigator" attribute>credentials</a>.<a for="CredentialsContainer" method lt="get()">get</a>({
+          ...,
+          <a for="CredentialRequestOptions" dict-member>mediation</a>: "<a for="CredentialMediationRequirement" enum-value>silent</a>"
+        });
+        if (c) {
+          // Hooray! Let's sign the user in using these credentials!
+        }
+      });
+    </pre>
+  </div>
+
+  <div class="example" id="example-mediation-optional">
+    When a user clicks "Sign In", MegaCorp, Inc. wishes to give them the smoothest possible
+    experience. If they have [[#user-mediation-requirement|opted-into]] signing in without
+    [=user mediation=], and the user agent can unambigiously choose a credential, great! If
+    not, a [=credential chooser=] will be presented.
+
+    <pre>
+      document.querySelector('#sign-in').addEventListener('click', e =&gt; {
+        var c = await navigatior.<a for="Navigator" attribute>credentials</a>.<a for="CredentialsContainer" method lt="get()">get</a>({
+          ...,
+          <a for="CredentialRequestOptions" dict-member>mediation</a>: "<a for="CredentialMediationRequirement" enum-value>optional</a>"
+        });
+        if (c) {
+          // Hooray! Let's sign the user in using these credentials!
+        }
+      });
+    </pre>
+
+    Note: MegaCorp, Inc. could also have left off the {{CredentialRequestOptions/mediation}}
+    member entirely, as "{{CredentialMediationRequirement/optional}}" is its default.
+  </div>
+
+  <div class="example" id="example-mediation-require">
+    MegaCorp, Inc. wishes to protect a sensitive operation by requiring a user to reauthenticate
+    before taking action. Even if a user has [[#user-mediation-requirement|opted-into]] signing in
+    without [=user mediation=], MegaCorp, Inc. can ensure that the user agent requires mediation
+    by calling {{get()}} with a {{CredentialRequestOptions/mediation}} member set to
+    "{{CredentialMediationRequirement/required}}":
+
+    <pre>
+      document.querySelector('#important-form').addEventListener('submit', e =&gt; {
+        var c = await navigatior.<a for="Navigator" attribute>credentials</a>.<a for="CredentialsContainer" method lt="get()">get</a>({
+          ...,
+          <a for="CredentialRequestOptions" dict-member>mediation</a>: "<a for="CredentialMediationRequirement" enum-value>required</a>"
+        });
+        if (c) {
+          // Verify that |c| enables access, and cancel the submission
+          // if it doesn't.
+        } else {
+          e.preventDefault();
+        }
+      });
+    </pre>
+  </div>
+
+  <div class="example" id="example-mediation-switch">
+    MegaCorp, Inc. wishes to support switching from one user account to another. In order to ensure
```

Comment:
"Switching from one user account to another" could be accomplished by logging out of the first, but so it might be clearer to say they wish to support logging into two accounts/identities at once.